configs/default/HBRO-KAKUTEF7.config

```diff
@@ -1,4 +1,5 @@
 # Betaflight / STM32F745 (S745) 4.0.0 Apr  3 2019 / 14:32:23 (22b9f3453) MSP API: 1.41
+#wiki https://github.com/betaflight/betaflight/wiki/Board---KAKUTEF7
```

Comment:
It’s great to have such links, but I think an update is called for. You still refer to 32K loop rate which we no longer support.

“F7 processor allows for faster looptime in theory without overclocking like we do with F4 running 32KHz”

Reply:
This is outside the scope of this config file update however.

Reply:
Yes documentation lacks updates.

Reply:
Well, I just made the possibility to refer to the documentation :-), not intented to update the documentation as well. :-)
But..., Holybro still sell the F7, so I will make an update of the documentation. I can also check  F4, also still in sale. :-)